framework/src/st_things/things_stack/things_def.h

```diff
@@ -51,4 +51,8 @@
 #define URI_DEVICE_COL                  "/device"
 #define URI_FIRMWARE                    "/firmware"
 
+#define DEVICE_OS_VERSION               "1.0"
```

Comment:
Is it used?

Reply:
Yes this is necessary.
I do not think user needs to enter this item.